app/assets/stylesheets/ingredients.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Ingredients controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
Let's remove redundant stuff that was automatically generated but is not used by us (this file)